docs/bounties/prices.md

```diff
@@ -0,0 +1,41 @@
+## Bounty Types
+
+Bounties can be classified into one of the following types:
+
+* 🐞 **Bug Fix**: Fix to a bug in the Hummingbot codebase
+* 🚀 **Enhancement**: Update or improvement to a component in the Hummingbot codebase
+* 📺 **Content**: Documentation, videos, guides, and other content that explains how to use an aspect of Hummingbot
+* 🔬 **Research**: Investigation into possible work and recommendation on how to proceed
+* 🏦 **New Connector**: A new [connector](/exchanges/) in the Hummingbot or the Hummingbot Gateway codebases
+* 📈 **New Strategy/Script**: A new Hummingbot [strategy](/strategies/) or [script](/scripts/)
+
+In addition, bounties may be:
+
+* **Open Source**: merged into the official Hummingbot codebase, or 
+* **Closed Source**: delivered as a private fork to the Sponsor
+
+## Suggested Bounty Amounts
+
+- Candles Feed: 500 USDC (625 USDC - 20% HB Foundation Fee)
```

Comment:
Here I would left this format:

Candles Feed: 625 USDC 

I think it's needed to not have much attention to the fees  (for me as a sponsor I will have a question why fees 20% etc.)

or we could change it to `Fee for handing`
Candles Feed: 625 USDC (developer portion: 500 USDC, Fee for handling: 125 USDC)

Personally I would go with the first (Candles Feed: 625 USDC) 
and add details to Github ticket when bounty funded

Reply:
we added it here:
https://www.notion.so/hummingbot-foundation/Bounty-price-24c275fd3853407a934bcaace36c172e

So could be something like 

![image](https://github.com/hummingbot/hummingbot-site/assets/83953535/7b48284f-ce8d-4371-84eb-1f29f0ee1845)